src/CommunityToolkit.Maui.Markup.UnitTests/Base/BaseTestFixture.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Globalization;
+using CommunityToolkit.Maui.Markup.UnitTests.Mocks;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Dispatching;
+using NUnit.Framework;
```

Comment:
Maybe we can rewrite tests using xUnit. But it is not necessary in this PR

Reply:
I'm not against it.

We re-write the `CommunityToolkit.Maui` Unit Tests in Xunit to match .NET MAUI's Unit Tests.

Xunit may also make it easier to write [Device Tests](https://github.com/dotnet/maui/tree/main/src/Core/tests/DeviceTests).